### ISSUE TITLE
Unable to determine Swift version for the following pods: admob_flutter

### ISSUE THREAD

**Issue by nemoryoliver** [YoussefKababe/admob_flutter#32]
Getting this error using v0.3.0

```
[!] Unable to determine Swift version for the following pods:

- `admob_flutter` does not specify a Swift version and none of the targets (`Runner`) integrating it have the `SWIFT_VERSION` attribute set. Please contact the author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod.
```

**Comment by nemoryoliver** [comment]
Fixed it by switching from Obj-C to Swift /ios folder

**Comment by kmcgill88** [comment]
Interesting, I always use swift in my projects so I didn't run into this. Easy enough fix to update the podspec. Glad you found a work around for now. 

PR's welcome! 😃 otherwise I'll get this added next chance I get.

**Comment by kmcgill88** [comment]
This should be fixed in 0.3.1. @nemoryoliver can you confirm? Then we can close this issue.

**Comment by nemoryoliver** [comment]
All good now! Thanks a lot
